### PR TITLE
challenge/isabel gomez game of life

### PR DESCRIPTION
**Pull request by IsabelGomez87**

### CODE REVIEW

**Review by jasuarez21 — APPROVED** [review]
buen trabajo ! isabeeeeeeeeeeeeeeeeel basta!

**Review by IrenePerezAlonso — APPROVED** [review]

File: Isabel-Gomez/challenge/game-of-life/game.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+const createNewMatrix = (row, column) => {
+    let newArray = new Array(row);
+    for (let i = 0; i < row; i++) {
+        newArray[i] = new Array(column).fill(0);
+    }
+    return newArray;
+};
+
+let matrix = createNewMatrix(5, 5);
+let newCells = [...matrix];
```
Comment (shd8) [inline Isabel-Gomez/challenge/game-of-life/game.js:10]: Better declare global variables at the top and then declare the functions.

File: Isabel-Gomez/challenge/game-of-life/game.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            } else if (array[rowIndex][cellIndex] === 0 && numOfNeighbours === 3) {
+                lastArray[rowIndex][cellIndex] = 1;
+            }
+        });
+    });
+    console.log('lastArray', lastArray);
+
+    return lastArray;
+}
+let getLastArray = checkEveryPosition(newCells);
```
Comment (shd8) [inline Isabel-Gomez/challenge/game-of-life/game.js:104]: Once more, better declare global variables at top.

**Review by shd8 — APPROVED** [review]
Good code, good tests, well done!

**Review by edalandete — APPROVED** [review]

File: Isabel-Gomez/challenge/game-of-life/game.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    }
+    if (array[row - 1]) {
+        array[row - 1][column] = 1;
+    }
+    if (array[row + 1]) {
+        array[row + 1][column] = 1;
+    }
+    return array;
+
+}
```
Comment (albertmasats) [inline Isabel-Gomez/challenge/game-of-life/game.js:24]: I suggest you to move that key to the line before. :)

File: Isabel-Gomez/challenge/game-of-life/game.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                changeCells(rowIndex, cellIndex, lastArray, numOfNeighbours);
+            } else if (array[rowIndex][cellIndex] === 0 && numOfNeighbours === 3) {
+                lastArray[rowIndex][cellIndex] = 1;
+            }
+        });
+    });
+    console.log('lastArray', lastArray);
+
+    return lastArray;
+}
```
Comment (albertmasats) [inline Isabel-Gomez/challenge/game-of-life/game.js:103]: line break here, just little changes, same as shd8 said, variables at top, const beffore let. :)

**Review by albertmasats — COMMENTED** [review]
Si haces los cambios te lo apruebo :)

**Pull request merged**